org.openhab.binding.zigbee/src/main/resources/OH-INF/thing/xiaomi/lumiremotemini.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="zigbee"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="xiaomi_lumiremotemini" listed="false">
+		<label>AQARA WXKG11LM</label>
+		<description>AQARA WXKG11LM</description>
```

Comment:
Please can you use a description of the device - not just repeat the label so that it's a bit more useful to users.